stub-packages/react-hooks/src/index.ts

```diff
@@ -0,0 +1,2 @@
+import './version';
+export * from '@uifabric/react-hooks';
```

Comment:
This index file should be the only export for react-hooks to match exports of the v8 version.